prebid-server/hosting/bidder-specific-guidelines.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page_v2
+title: Bidder-Specific Guidelines for PBS Host Companies
+description: Bidder-Specific Guidelines for PBS Host Companies
+pid: 0
+sidebarType: 5
+---
+
+# Bidder-Specific Guidelines for PBS Host Companies
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Setting up a Bid Adapter
+
+In general, a PBS host company may want to run only the server-side bid adapters they require.
+
+PBS-Go enables most bidders by default, while PBS-Java doesn't
+enable any bidders out of the box.
+
+### PBS-Go
+
+You might want to consider disabling bid adapters that you're not 
+going to utilize in order to control which bidders get into the [/cookie_sync](/prebid-server/endpoints/pbs-endpoint-cookieSync.html) response when you're running the `coopSync` flag.
+
+TBD: how would a host company disable bidders?
```

Comment:
To disable a bid adapter, in your main pbs.yaml file, set:

```
adapters:
  BIDDERCODE:
    disabled: false
```

If the bidder supports different endpoints per geography, you can deploy different
config in each of your datacenters:

```
adapters:
  BIDDERCODE:
    endpoint: REGION_SPECIFIC_ENDPOINT
```

Reply:
PBS-Go also supports json, toml, hcl, envfile, java properties, and environment variables config file formats. For simplicity, let's just include the yaml example.